README.md

```diff
@@ -158,7 +158,6 @@ Procuramos ordenar os conteúdos e criar uma navegação amigável, visando faci
 - [Fabio Akita](https://www.youtube.com/user/AkitaOnRails) - Canal aborda a experiência pessoal do Fabio Akita, que trabalha na área há décadas, sua visão sobre linguagens, metodologias de estudo, questões sobre a área em geral e Blockchain. _Tags: `ruby on rails`, `carreira em ti`_
 - [Fernanda Bernardo](https://www.youtube.com/user/nandinhabernardo) - O canal da Fernanda Bernardo está começando com uma série maravilhosa sobre comunicação, onde os assuntos que são tratados lá no @Help4Papers estão se tornando ótimos videos. Se você quer palestrar ou aprender a se comunicar melhor, assina agora o canal. _Tags: `carreira em ti`_
 - [Ferreira Studios](https://www.youtube.com/channel/UCztMwfGqsnPeGat7Np-bFKg) - Canal de desenvolvimento web, dicas e métodos sobre design e ferramentas criativas, do designer gráfico e desenvolvedor Leonardo Ferreira. _Tags: `design`_
-- [Filho da nuvem](https://www.youtube.com/filhodanuvem) - Um canal para ajudar semanalmente pessoas a dar o próximo passo na programação. Como criar projetos no GitHub, estudos sobre a linguagem Go e carreira em Portugal são alguns dos temas. _Tags `carreira em ti`, `golang`_
```

Comment:
@viktorleite Não entendi qual link esta duplicado.

Reply:
> @viktorleite Não entendi qual link esta duplicado.

É que o canal **Filhos da nuvem** esta nos tópicos _Desenvolvimento Back-end_ e _Entrevista, Webinars & Dicas_

Reply:
Ah verdade, você consegue criar um teste para impedir links duplicados na lista? Se não for possível, tranquilo.. mas me ajudaria se fizesse 😀

Reply:
> Ah verdade, você consegue criar um teste para impedir links duplicados na lista? Se não for possível, tranquilo.. mas me ajudaria se fizesse

O script _**readme_checker_test.rb**_ agiliza esse processo quando fazemos uma **Pull Request*.

Reply:
Sim, eu sei hahahahahaha 
Mas não deveria permitir links duplicados, provavelmente este teste não esta coberto no script.

Reply:
Acabei de ver que o teste já existe e deveria cobrir este cenário. Acho que foi algum problema na hora de fazer o merge que acabou duplicando esse link